pegasus/sites.v3/code.org/public/educate/applab.haml

```diff
@@ -25,21 +25,24 @@ video_player: true
   .col-50
     .tutorial-tile
       %a{:href=>CDO.studio_url("/s/applab-intro/reset")}
-        %img.tutorial-tile-img{src: CDO.code_org_url("/images/app-lab/applab-intro.png"), style: "width: 100%;"}
+        %img.tutorial-tile-img{src: CDO.code_org_url("/images/app-lab/applab-intro.png")}
       .tutorial-info
         %h3.tutorial-info-h Intro to App Lab Tutorial (beta)
         .smalltext Create your own app in JavaScript using either block based programming or text. If you've already done some programming with blocks, take your skills to the next level.
         %a{:href=>CDO.studio_url("/s/applab-intro/reset")}
-          %button.tutorial-gray{style: "margin-top: 20px;"} Start tutorial
-        %div{style: "float: right; margin-top: 25px;"}
+          %button.tutorial-info-start.tutorial-gray Start tutorial
+        .tutorial-info-guide
           %a{href: "https://curriculum.code.org/hoc/plugged/7/"}
             View teacher guide
+        .clear
```

Comment:
Adding this prevents content from flowing outside the box down at 320px width screens.